src/main/java/frc/core/util/TrajectoryBuilder.java

```diff
@@ -59,6 +56,16 @@ public TrajectoryBuilder(Drivetrain drivetrain, String... filesNames) {
 	}
 
 	//helpers
+	public void configTrajectories(String... filesName){
+		this.trajectories = new HashMap<>();
+
+		Arrays.stream(filesName)
```

Comment:
bruno, tem uma maneira mais eficiente de criar um map com programação funcional.
Segue o exemplo:

this.trajectories = Arrays.stream(filesNames) // transforma o array em um stream
			    .collect(Collectors.toMap( // diz que irá coletar esse array como um map
				    fileName -> fileName, // tu escolhe o primeiro parametro desse map (key)
				    fileName -> this.createTrajectory(fileName) // segundo parametro (value)
			    ));